tests/src/test/scala/system/basic/WskBasicSwift3Tests.scala

```diff
@@ -33,19 +33,20 @@ import common.TestUtils.RunResult
 import spray.json.JsObject
 
 @RunWith(classOf[JUnitRunner])
-class WskBasicSwiftTests
+class WskBasicSwift3Tests
     extends TestHelpers
     with WskTestHelpers
     with JsHelpers {
 
     implicit val wskprops = WskProps()
     val wsk = new Wsk
     val defaultAction = Some(TestUtils.getTestActionFilename("hello.swift"))
-    val currentSwiftDefaultKind = "swift:3"
+    lazy val currentSwiftDefaultKind = "swift:3"
 
     behavior of "Swift runtime"
 
-    it should "Map a kind of swift:default to the current default swift runtime" in withAssetCleaner(wskprops) {
+    // ignore this while deprecated swift:3 is still default
+    ignore should "Map a kind of swift:default to the current default swift runtime" in withAssetCleaner(wskprops) {
```

Comment:
Ah, I remember now.  This test gets inherited by WskBasicSwift311Tests and will fail.  At the time I thought updating the default to 3.1.1 was imminent.